models/nlp/albert/run_pretraining.py

```diff
@@ -66,9 +66,13 @@
 # See https://github.com/huggingface/transformers/issues/3782; this import must come last
 import smdistributed.dataparallel.tensorflow as smddp  # isort:skip
 
-import smddpcommon
+# TODO : Change to obfuscate smddpcommon. This code does not use GradientTape, so need to pass it like this.
 bucket_cap_bytes = int(64 * 1024 * 1024)
-smddpcommon.setBucketSize(bucket_cap_bytes) # TODO : Change to obfuscate smddpcommon. This code does not use GradientTape, so need to pass it like this.
```

Comment:
we are still using smddpcommon/herringcommon and thus the comment still remains valid. why are we removing it?